msg/estimator_innovations.msg

```diff
@@ -0,0 +1,41 @@
+uint64 timestamp		# time since system start (microseconds)
+
+# GPS
+float32[2] gps_hvel	# horizontal GPS velocity innovation (m/sec) and innovation variance ((m/sec)**2)
+float32    gps_vvel	# vertical GPS velocity innovation (m/sec) and innovation variance ((m/sec)**2)
+float32[2] gps_hpos	# horizontal GPS position innovation (m) and innovation variance (m**2)
+float32    gps_vpos	# vertical GPS position innovation (m) and innovation variance (m**2)
+
+# External Vision
+float32[2] ev_hvel	# horizontal external vision velocity innovation (m/sec) and innovation variance ((m/sec)**2)
+float32    ev_vvel	# vertical external vision velocity innovation (m/sec) and innovation variance ((m/sec)**2)
+float32[2] ev_hpos	# horizontal external vision position innovation (m) and innovation variance (m**2)
+float32    ev_vpos	# vertical external vision position innovation (m) and innovation variance (m**2)
+
+# Fake Position and Velocity
+float32[2] fake_hvel	# fake horizontal velocity innovation (m/s) and innovation variance ((m/s)**2)
+float32    fake_vvel	# fake vertical velocity innovation (m/s) and innovation variance ((m/s)**2)
+float32[2] fake_hpos	# fake horizontal position innovation (m) and innovation variance (m**2)
+float32    fake_vpos	# fake vertical position innovation (m) and innovation variance (m**2)
+
+# Height sensors
+float32 rng_vpos	# range sensor height innovation (m) and innovation variance (m**2)
+float32 baro_vpos	# barometer height innovation (m) and innovation variance (m**2)
+
+# Auxiliary velocity
+float32[2] aux_hvel	# horizontal auxiliar velocity innovation from landing target measurement (m/sec) and innovation variance ((m/sec)**2)
+float32    aux_vvel	# vertical auxiliar velocity innovation from landing target measurement (m/sec) and innovation variance ((m/sec)**2)
+float32[2] flow		# flow innvoation (rad/sec) and innovation variance ((rad/sec)**2)
+
+# Various
+float32 heading		# heading innovation (rad) and innovation variance (rad**2)
```

Comment:
What if I have multiple heading sources? Compass, GPS Yaw and VIO are some options.

Reply:
That is a good observation. That can be tackled when someone is working on multi sensor heading fusion.

Reply:
The thing is, this message is not easily changeable in the future, and every change is a breaking one due to the log parsing tools.

Reply:
One solution could be adding now fields such that we would have [gps_heading, ev_heading, mag_heading, heading] and only use the heading field. And after adding the future changes still fill the heading field in addition to all the other, but mark the heading field as deprecated. This would give log users time to adopt. What do you think about that approach?

Reply:
I would prefer adding the additional and at the moment unused heading field at the moment when we at the functionality.